### PR TITLE
Add aardio

### PR DESCRIPTION
**Pull request by theseazhang**
## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme

### CODE REVIEW

**Comment by MrBrain295** [comment]
You need to add this to the readme. Also I need a link to the languages GitHub repo or website.

**Comment by Richienb** [comment]
Could you provide a link to the programming language?

**Comment by Richienb** [comment]
Not sure if the name should be capitalised

File: a/aardio.aardio [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import console
+console.log("hello world")
```
Comment (MrBrain295) [inline a/aardio.aardio:2]:
```suggestion
console.log("Hello World")
```

**Review by MrBrain295 — CHANGES_REQUESTED** [review]
The readme also needs updated.

**Pull request closed without merging**